pages/common/ab.md

```diff
@@ -1,24 +1,24 @@
 # ab
 
-> Apache Benchmarking tool. The simplest tool to perform a load testing.
-> More information: <https://httpd.apache.org/docs/2.4/programs/ab.html>.
+> Apache HTTP server benchmarking tool.
+> More information: <https://httpd.apache.org/docs/current/programs/ab.html>.
 
-- Execute 100 HTTP GET requests to a given URL:
+- Execute 100 HTTP GET requests to a URL:
```

Comment:
No need to remove `given` here. In fact, it sounds more grammatically correct if you include it.

Reply:
Hmm, maybe so.
```suggestion
- Execute 100 HTTP GET requests to a given URL:
```